apps/angular/src/app/features/auth/models/login-credentials.ts

```diff
@@ -0,0 +1,4 @@
+import { RegisterCredentials } from './register-credentials';
+
+/** Login credentials. */
+export type LoginCredentials = Pick<RegisterCredentials, 'email' | 'password'>;
```

Comment:
I don’t think that the login and registration model are in any way related to each other.
```suggestion
export type LoginCredentials = {

	/** Email. */
	readonly email: string;

	/** Password. */
	readonly password: string;

};
```